src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.scss';
+import {
+  AddToDoList,
+  setTasks,
+  showTasks,
+  GetToDoList,
+  updateLocalStorage,
+} from './modules/AddToDoItems.js';
+import { clearAllCompletedTasks } from './modules/Completed.js';
+
+const newTodolistInput = document.querySelector('#add-to-do > input');
```

Comment:
- [ ] As mentioned by the previous reviewer, correct element to trigger the add operation is not selected. You might need to console.log which element you are referencing to. So you would know whether you're selecting the right element. At this moment, I am unable to add task to todo-list. Kindly look into that and fix. 
When I checked the last you made, you only changed the content of the title. Please, take a look. :point_down: 
![your-commit](https://user-images.githubusercontent.com/26597432/222916879-bd50688b-975a-4e30-a2d8-bbba5e7a5505.png)

### You app behavior
[app-behaviour.webm](https://user-images.githubusercontent.com/26597432/222916901-3c576a96-3860-412a-aa3d-65bc74db32d6.webm)